content/cloud-docs/workspaces/creating.mdx

```diff
@@ -34,48 +31,49 @@ New workspaces can be created by teams with permission to manage workspaces. ([M
+For CLI and API-driven workspaces, the workspace overview page appears. For version control workspaces, the **Configure Terraform variables** page appears.
 
-   -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
+### Configure Terraform Variables (Version Control Only)
 
-1. On the third screen, enter a name for the workspace. This defaults to the repository name, if applicable. The name must be unique within the organization, and can include letters, numbers, dashes (`-`), and underscores (`_`). See also our [advice for useful workspace names](/cloud-docs/workspaces/naming).
+After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and automatically creates those variables within the workspace. Terraform cannot perform successful runs until you set values for these undefined variables.
 
-1. Optionally, click the "Advanced options" link on the third screen to configure some additional version control settings. (These settings not shown if you chose "No VCS connection".) For information about these settings, see:
-   - [Terraform Working Directory][workdir]
-   - [Automatic Run Triggering][trigger]
-   - [VCS branch][branch]
-   - [Include submodules on clone][submodules]
+Choose one of the following actions:
 
-1. Confirm creation with the "Create workspace" button.
+- To skip this step, click **Go to workspace overview**. Terraform Cloud still adds any undefined Terraform variables without default values to the workspace. You can set values for these variables later from within the workspace.
+- To configure variables, enter a value for each variable and click **Save variables**.
 
 ## After Creating a Workspace
 
-Terraform Cloud presents a dialog with shortcut links to either queue a plan or edit variables. If you don't need to edit variables, [manually queue a run](/cloud-docs/run/ui#manually-starting-runs) to prepare your workspace.
-
-You may also want to:
+If you have already configured all Terraform variables, we recommend [manually starting a run](/cloud-docs/run/ui#manually-starting-runs) to prepare VCS-driven workspaces. You may also want to do one or more of the following actions:
 
+- [Upload configuration versions](/cloud-docs/workspaces/configurations#providing-configuration-versions): If you chose the API or CLI-Driven workflow, you must upload configuration versions for the workspace.
 - [Edit input or environment variables](/cloud-docs/workspaces/variables): Input variables define the parameters of a Terraform configuration, and shell environment variables store credentials and customize Terraform's behavior.
```

Comment:
I think you can remove the bits in this item about input variables -- since those are handled above with line 63.

```suggestion
- [Edit environment variables](/cloud-docs/workspaces/variables): Shell environment variables store credentials and customize Terraform's behavior.
```